src/test/common/clients/featureTogglesClient.ts

```diff
@@ -0,0 +1,35 @@
+import { expect } from 'chai'
+import * as featureToggleApiMock from 'test/http-mocks/feature-toggle-store'
+import { FeatureTogglesClient } from 'main/common/clients/featureTogglesClient'
+import { User } from 'main/app/idam/user'
+import * as config from 'config'
+import 'test/routes/expectations'
+
+const serviceBaseURL: string = config.get<string>('feature-toggles-api.url')
+const featureTogglesClient: FeatureTogglesClient = new FeatureTogglesClient(serviceBaseURL)
+const roles: string[] = []
+const user: User = new User('','','','',roles,'','')
+
+describe('featureToggleClient', () => {
+  describe('when ff4j is available', () => {
+    it('should return admission is off when feature toggle for admission is disabled', async () => {
+
+      featureToggleApiMock.resolveIsAdmissionsAllowed(false)
+      expect(await featureTogglesClient.isAdmissionsAllowed(user, roles)).to.be.eqls(false)
```

Comment:
Should be `.to.be.false`
will need to add an instruction to tslint to ignore it